backend/FwLite/MiniLcm/Validators/CanonicalGuidsPartOfSpeech.cs

```diff
@@ -0,0 +1,69 @@
+﻿using FluentValidation;
+
+namespace MiniLcm.Validators;
+
+public static class CanonicalGuidsPartOfSpeech
+{
+    // GUID list taken from src/SIL.LCModel/Templates/GOLDEtic.xml in liblcm
+    // TODO: Consider loading GOLDEtic.xml into app as a resource and add singleton providing access to it, then look up GUIDs there rather than using this hardcoded list
+    public static HashSet<Guid> CanonicalPosGuids = [
```

Comment:
this should be readonly, right now the field is mutable, and the HashSet is mutable. Obviously make the field readonly, instead I'd use a `FrozenSet<T>`

Reply:
Done.